### PR TITLE
add support for multiple pallets in a freight shipment

### PR DESCRIPTION
**Pull request by aastronautss**

### CODE REVIEW

File: lib/fedex/request/base.rb [hunk 1]
```diff
@@ -219,7 +228,7 @@ def add_packages(xml)
       
       def add_freight_shipment_detail(xml)
         xml.FreightShipmentDetail {
-          if shipping_from_our_warehouse?
+          # if shipping_from_our_warehouse?
```
Comment (zreisman) [inline lib/fedex/request/base.rb]: Is this related?

Comment (aastronautss) [reply lib/fedex/request/base.rb]: oh yeah, i need to just remove this all completely. basically, we're always going to be billing collect when it's inbound, and collect is incompatible with an alternate billing method.

Comment (aastronautss) [reply lib/fedex/request/base.rb]: (also all this stuff should go into a wrapper for the main repo instead of here)
